ui/panels-plugin/src/model/thresholds.ts

```diff
@@ -11,53 +11,38 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-import { UnitOptions } from '@perses-dev/components';
+import { UnitOptions, ThresholdColorPalette } from '@perses-dev/components';
+import { StepOptions, ThresholdOptions } from '@perses-dev/plugin-system';
 import { zip } from 'lodash-es';
 
-// TODO (sjcobb): pull threshold colors from perses charts theme
-export const ThresholdColors = {
-  GREEN: 'rgba(47, 191, 114, 1)', // green.500
-  YELLOW: 'rgba(255, 193, 7, 1)',
-  ORANGE: 'rgba(255, 159, 28, 0.9)', // orange.500
-  RED: 'rgba(234, 71, 71, 1)', // red.500
-};
-
-export const ThresholdColorsPalette = [ThresholdColors.ORANGE, ThresholdColors.RED];
-
-export type ThresholdColorsType = keyof typeof ThresholdColors | string;
-
 export type GaugeColorStop = [number, string];
 
 export type EChartsAxisLineColors = GaugeColorStop[];
 
-export interface StepOptions {
```

Comment:
moved `StepOptions` and `ThresholdOptions` to plugin-system since I needed these types in `ThresholdsEditor` which lives in plugin-system/components

Reply:
Another option could be to move ThresholdsEditor to @perses-dev/components and the types to @perses-dev/core.

We definitely need to decide on long term rules for what goes in each package since currently it isn't consistent (see [discussion here](https://github.com/perses/perses/pull/761#discussion_r1022875646): LegendOptionsEditor, UnitSelector, unit types are in @perses-dev/components for instance). Probably fine to wait until @juliepagano looks into package reorganization though too

Reply:
I think ThresholdsEditor being in @perses-dev/components will also make it easier to have context-aware components in the future. If it lives in plugin-system, then ColorPicker.tsx would need to move to avoid a circular dependency for instance

Reply:
i think i was debating to move it to @perses-dev/components but got confused since some components like CalculationSelector are in plugin-system. Can't wait to get this figure out lol.

Anyways moved ThresholdsEditor to @perses-dev/component 😄